Mlem/App/Views/Root/Tabs/Settings/DeveloperSettingsView.swift

```diff
@@ -16,10 +17,24 @@ struct DeveloperSettingsView: View {
     
     var body: some View {
         Form {
-            Toggle(String("Developer Mode"), isOn: $developerMode)
-            Button(String("Reset Feed Welcome Prompt")) {
-                showFeedWelcomePrompt = true
+            Section {
+                Toggle(String("Developer Mode"), isOn: $developerMode)
+                NavigationLink("Error Log", destination: .settings(.errorLog))
```

Comment:
Currently we aren't localizing anything in developer settings, which by extension includes the error log.

```suggestion
                NavigationLink(String("Error Log"), destination: .settings(.errorLog))
```